tests/Application/config/packages/sylius_invoicing.yaml

```diff
@@ -0,0 +1,23 @@
+imports:
+    - { resource: "@SyliusInvoicingPlugin/Resources/config/config.yml" }
+            
+sylius_invoicing:
```

Comment:
Why is this necessary for tests?

Reply:
Without it you can't test that the entities are overridable, do you have another better solution ?